client/cody/src/status-bar.ts

```diff
@@ -0,0 +1,99 @@
+import * as vscode from 'vscode'
+
+import type { Configuration } from '@sourcegraph/cody-shared/src/configuration'
+
+import { getConfiguration } from './configuration'
+
+export interface CodyStatusBar {
+    dispose(): void
+    startLoading(label: string): () => void
+}
+
+export function createStatusBar(): CodyStatusBar {
+    const statusBarItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right)
+    statusBarItem.text = '$(cody-logo)'
```

Comment:
```suggestion
    statusBarItem.tooltip = 'Cody Features Toggle'
    statusBarItem.name = 'Cody AI by Sourcegraph'
    statusBarItem.text = '$(cody-logo)'
```

Before: 
No hover test:
<img width="675" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/03075c42-5f37-41f9-8d23-2d926df55055">

<img width="1144" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/dcb49e93-d3de-4b3b-a738-63379e1c735d">


After:
<img width="458" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/9ef80dea-c377-4bf4-a9ce-787a63fbb977">
<img width="803" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/96b4a096-c659-49e4-93e7-efac4ace8d8d">

Reply:
Also makes sense, thanks! 😎